Libs/DICOM/Widgets/Testing/Cpp/CMakeLists.txt

```diff
@@ -55,7 +64,9 @@ SIMPLE_TEST(ctkDICOMModelTest2
   ${CMAKE_CURRENT_BINARY_DIR}/dicom.db
   ${CMAKE_CURRENT_SOURCE_DIR}/../../../Core/Resources/dicom-sample.sql
   )
+if(CTK_QT_VERSION VERSION_GREATER "4")
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.